cmd/kubeadm/app/phases/controlplane/volumes.go

```diff
@@ -37,11 +37,6 @@ const (
 	kubeConfigVolumeName = "kubeconfig"
 )
 
-// caCertsPkiVolumePath specifies the path that can be conditionally mounted into the apiserver and controller-manager containers
-// as /etc/ssl/certs might be a symlink to it. It's a variable since it may be changed in unit testing. This var MUST NOT be changed
-// in normal codepaths during runtime.
-var caCertsPkiVolumePath = "/etc/pki"
```

Comment:
It looks like the unit tests that used this variable need a fix ([test log](https://k8s-gubernator.appspot.com/build/kubernetes-jenkins/pr-logs/pull/49840/pull-kubernetes-unit/44682/?log#log)): 
```
[...]
W0804 00:30:36.473] # k8s.io/kubernetes/cmd/kubeadm/app/phases/controlplane
W0804 00:30:36.474] cmd/kubeadm/app/phases/controlplane/volumes_test.go:276: not enough arguments in call to getEtcdCertVolumes
W0804 00:30:36.474] 	have (kubeadm.Etcd)
W0804 00:30:36.475] 	want (kubeadm.Etcd, string)
W0804 00:30:36.475] cmd/kubeadm/app/phases/controlplane/volumes_test.go:514: undefined: caCertsPkiVolumePath
W0804 00:30:36.475] cmd/kubeadm/app/phases/controlplane/volumes_test.go:515: undefined: caCertsPkiVolumePath
[...]
```